src/storm-dft/parser/BEOrderParser.cpp

```diff
@@ -0,0 +1,58 @@
+#include "BEOrderParser.h"
```

Comment:
I think we have a habit of writing includes differently. I would suggest to also do that for consistency (although I don't really mind...)

Reply:
Done.